charts/kyverno/README.md

```diff
@@ -38,7 +38,7 @@ $ kubectl create namespace kyverno
 **Install the Kyverno chart:**
 
 ```console
-$ helm install kyverno --namespace kyverno kyverno ./charts/kyverno
+$ helm install kyverno --namespace kyverno kyverno kyverno/kyverno
```

Comment:
i think it should be `helm install kyverno --namespace kyverno kyverno/kyverno`

Reply:
Good catch, fixed.